doc/releases/changelog-dev.md

```diff
@@ -296,6 +296,9 @@
   and the diagonalising gates using the factors/summands instead of using the full matrix.
   [(#3022)](https://github.com/PennyLaneAI/pennylane/pull/3022)
 
+* `qml.grouping.is_pauli_word` now returns `False` for non-Observables, instead of raising an error.
```

Comment:
Just a little more specific 
```suggestion
* `qml.grouping.is_pauli_word` now returns `False` for operators that don't inherit from `qml.Observable`, instead of raising an error.
```

Reply:
The following would fail this test and still be considered a Pauli word:   ` (1/sqrt(2)) * qml.op_sum(qml.PauliX(0), qml.PauliZ(0))`